goetas/twital-bundle/1.2/config/twital.yaml

```diff
@@ -0,0 +1,10 @@
+framework:
+    templating:
+        engines: ['twig', 'twital']
+# optional configurations for file extension matching
```

Comment:
Not sure we should include this optional config. I would much rather see a link to the bundle's documentation